src/axom/primal/operators/detail/intersect_bezier_impl.hpp

```diff
@@ -130,7 +232,12 @@ bool intersect_bezier_curves(const BezierCurve<T, 2> &c1,
 
   bool foundIntersection = false;
 
-  if(c1.isLinear(sq_tol) && c2.isLinear(sq_tol))
+  if(c1.isLinear(sq_tol))
+  {
+    c1.isLinear(sq_tol, true);
+  }
```

Comment:
This doesn't look right.  Possible weird merge?
I think the inner call just returns a bool.

Should these four lines be removed?

Reply:
Yep, should absolutely be removed. Old debugging statements. Sorry!